.github/workflows/django-multitenant-tests.yml

```diff
@@ -4,6 +4,8 @@ on:
     branches:
       - "**"
   workflow_dispatch:
+  pull_request:
+    types: [ opened, reopened, synchronize ]
```

Comment:
Branches will only trigger in the main repository. For a PR like this one, which was created by a fork, we need to have a `pull_request` trigger to see the results in the PR. Is there another solution so that fork-based PRs will initiate a workflow?